projects/public-patron-profile/src/app/app-config-service.service.ts

```diff
@@ -38,6 +38,7 @@ export class AppConfigService extends CoreConfigService {
     this.production = environment.production;
     this.apiBaseUrl = environment.apiBaseUrl;
     this.$refPrefix = environment.$refPrefix;
+    this.schemaFormEndpoint = '/schemas';
```

Comment:
Is it used?

Reply:
Yes. here: `this._recordService.getSchemaForm('users')`